spec/lib/api/api_config_spec.rb

```diff
@@ -22,11 +22,19 @@
 
     describe 'identifiers' do
       let(:api_feature_identifiers) do
-        feature_identifiers { |set, id| set.add(id) }
-      end
+        collection_settings.each_with_object(Set.new) do |(_key, config), ids|
+          action_types = config.keys.select { |k| k.to_s.end_with?("_actions") }
+          action_identifiers = action_types.each do |action_type|
+            config[action_type].each do |_verb, actions|
+              actions.map do |action|
+                ids.merge(Array(action[:identifier]))
+                ids.merge(Array(action[:identifiers]).map { |i| i[:identifier] }.compact)
```

Comment:
This effectively fixes the bug in the test...it never checked `:identifiers` (plural) before.